### PR TITLE
feat: auto hide or show editor

### PR DESCRIPTION
**Pull request by YuJianghao**
feat: #594

- hide the editor when scrolling down fast enough.
- show the editor when scrolling up fast enough.

https://user-images.githubusercontent.com/17293034/205955671-3e8511d9-283d-43dc-b1cf-68828edc3e74.mov

### CODE REVIEW

**Comment by hyoban** [comment]
#697

File: web/src/less/memos-header.less [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .section-header-container,
 .memos-header-container {
-  @apply sticky top-4 flex flex-row justify-between items-center w-full h-10 flex-nowrap mt-4 mb-2 shrink-0 z-10;
+  @apply top-4 flex flex-row justify-between items-center w-full h-10 flex-nowrap mt-4 mb-2 shrink-0 z-10;
```
Comment (hyoban) [inline web/src/less/memos-header.less:3]: Is this necessary?

**Comment by boojack** [comment]
Thanks for the PR. I removed the sticky style for editor in https://github.com/usememos/memos/pull/752. It's scrollable and will not hide the content as same as Twitter.

File: web/src/less/memos-header.less (hunk 1 above)
Comment (YuJianghao) [reply web/src/less/memos-header.less:3]: yes

**Pull request closed without merging**